src/pages/home/report/ReportActionCompose.js

```diff
@@ -662,20 +679,53 @@ class ReportActionCompose extends React.Component {
     }
 
     /**
-     * Callback for the emoji picker to add whatever emoji is chosen into the main input
-     *
-     * @param {String} emoji
+     * Callback to add whatever text is chosen into the main input (used f.e as callback for the emoji picker)
+     * @param {String} text
      */
-    addEmojiToTextBox(emoji) {
-        this.updateComment(ComposerUtils.insertText(this.comment, this.state.selection, `${emoji} `));
+    replaceSelectionWithText(text) {
+        this.updateComment(ComposerUtils.insertText(this.comment, this.state.selection, `${text} `));
```

Comment:
Extra space is added from here, caution when modifying as we need space at the end for emoji but not for the text.

Reply:
Thanks for pointing this place out